rules.d/60-cryptsetup-id.rules

```diff
@@ -0,0 +1,15 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="cryptsetup_id_end"
+SUBSYSTEM!="block", GOTO="cryptsetup_id_end"
+
+KERNEL=="dm-*", IMPORT{program}="cryptsetup_id"
+
+ENV{CRYPT_TYPE}=="", GOTO="cryptsetup_id_end"
+
+ENV{CRYPT_DEVICE_SYSNAME}!="", SYMLINK+="disk/by-crypt-device/$env{CRYPT_DEVICE_SYSNAME}"
+ENV{CRYPT_METADATA_DEVICE_SYSNAME}!="", SYMLINK+="disk/by-crypt-metadata-device/$env{CRYPT_METADATA_DEVICE_SYSNAME}"
+ENV{CRYPT_DATA_DEVICE_SYSNAME}!="", SYMLINK+="disk/by-crypt-data-device/$env{CRYPT_DATA_DEVICE_SYSNAME}"
+ENV{CRYPT_HASH_DEVICE_SYSNAME}!="", SYMLINK+="disk/by-crypt-hash-device/$env{CRYPT_HASH_DEVICE_SYSNAME}"
```

Comment:
What is the use case of these symlinks?

Reply:
I do not need it. I need from a filesystem, to be able to find the hardware it is on.

But in my mind, it is as valid to go from hardware to filesystems. And the symlinks can help to find device mappers where a block device is active in.

I will remove this as well as the _SYSNAME variables. And we can add that in another PR.